src/types/onyx/ReportAction.ts

```diff
@@ -139,10 +139,10 @@ type ReportActionBase = OnyxCommon.OnyxValueWithOfflineFeedback<{
     created: string;
 
     /** report action message */
-    message?: Message[];
+    message?: Array<Message | undefined>;
```

Comment:
```suggestion
    message?: Message[];
```

Why this change? The code looks fine if I revert (along wit the other fixes)